config/initializers/okcomputer.rb

```diff
@@ -2,4 +2,4 @@
 
 OkComputer::Registry.register 'solr', OkComputer::SolrCheck.new(Blacklight.default_index.connection.uri.to_s.sub(/\/$/, ''))
 
-OkComputer::Registry.register 'version', OkComputer::AppVersionCheck.new(env: 'GIT_INFO')
+OKComputer::Registry.register 'redis', OkComputer::RedisCheck.new(url: ENV.fetch('SIDEKIQ_REDIS_URL', 'redis://localhost:6379/'))
```

Comment:
As this is a load balanced system do we want every node to run a check plus the check against the redis server itself? That seems like more checks than necessary.

Reply:
cc @corylown — I think you wanted a redis okcomputer check specifically?

Reply:
Since there's a check already in Nagios we probably don't need another one from the app itself.